core/src/main/scala/kafka/server/ConfigAdminManager.scala

```diff
@@ -69,14 +68,14 @@ import scala.jdk.CollectionConverters._
  * - The second step is "persistence," and handles storing the configurations durably to our
  * metadata store.
  *
- * When KIP-590 forwarding is active (such as in KRaft mode), preprocessing will happen
+ * When KIP-590 forwarding is active, preprocessing will happen
```

Comment:
That comment does not make sense anymore, KIP-590 is always "active" now that ZooKeeper has been removed.